### PR TITLE
Rename NewRange() to NewReplica()

### PR DESCRIPTION
**Pull request by kkaneda**

### CODE REVIEW

**Comment by bdarnell** [comment]
LGTM

File: storage/replica_command.go [hunk 1]
```diff
@@ -986,7 +986,7 @@ func (r *Replica) splitTrigger(batch engine.Engine, split *proto.SplitTrigger) e
 	// Add the new split range to the store. This step atomically
 	// updates the EndKey of the updated range and also adds the
 	// new range to the store's range map.
```
Comment (tschottdorf) [inline storage/replica_command.go]: comment could also use s/range/replica/i

Comment (kkaneda) [inline storage/replica_command.go]: Sure, fixed the comment.

Comment (kkaneda) [inline storage/replica_command.go]: Sorry, I forgot I already merged the PR. Will send a small PR to update the comment.

**Pull request merged**